README.md

```diff
@@ -19,14 +19,17 @@ Commands like `kubectl` allow you to export command completion logic for use in
 
 # Usage
 
-1. Locate the completion script for the bash command. You may have to export this like `kubectl completion bash > kubectl_completions.sh`
-2. Run the `Register-BashArgumentCompleter` cmdlet to register the command you're expanding and the location of the completions.
-
-Example:
+1. Locate the completion script for the bash command. You may have to export this like:
+    ```PowerShell
+    kubectl completion bash | Out-File -Encoding ASCII -NoNewline -FilePath kubectl_completions.sh
+    ```
+    **Make sure the completion file is UTF-8 or ASCII.** Some exports (like `kubectl`) come out as UTF-16. Windows `bash` may see this as a binary file that can't be interpreted which results in no completions happening.  Using the `Out-File` PowerShell cmdlet allows you to specify the encoding to ensure the completions script will load in bash.
```

Comment:
If it really needs to be ASCII, this probably shouldn't be "UTF-8 or ASCII," right? Would you mind grabbing that one, too?

Reply:
There are even more [complicated ways of creating UTF8 sans BOM](https://stackoverflow.com/questions/5596982/using-powershell-to-write-a-file-in-utf-8-without-the-bom) but those are kind of a pain when the command completions are safely ASCII.  In the future with PowerShell 6 we could switch to using `Set-Content` with UTF8NOBOM encoding.